README.md

```diff
@@ -4,8 +4,10 @@ A Docker container that runs [pa11y-dashboard](https://github.com/pa11y/dashboar
 
 ## Development
 Build and run with `./scripts/start`:
```

Comment:
looks like an extra ':' you don't need at the end of the line

Reply:
Gone...